buildWithHtsjdkSrc.gradle

```diff
@@ -0,0 +1,172 @@
+import javax.tools.ToolProvider
```

Comment:
I'm not sure I understand why this is a separate build file.  It seems like it should be a different configuration in the main build file.